src/Tickets/Emails/Email/RSVP.php

```diff
@@ -327,13 +341,21 @@ public function get_settings_fields(): array {
 	 */
 	public function get_content( $args = [] ): string {
 		// @todo: Parse args, etc.
-		$context = ! empty( $args['context'] ) ? $args['context'] : [];
+		$is_preview = ! empty( $args['is_preview'] );
```

Comment:
Would it be better to use `tribe_is_truthy()` here, in case a string is passed, like '0'?